src/index.js

```diff
@@ -21,7 +21,8 @@ for (const file of eventFiles) {
   const event = require(`./events/${file}`);
   if (event.once) {
     client.once(event.name, (...args) => event.execute(...args));
-  } else {
+  }
+ else {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 1 space.](https://app.codacy.com/gh/ZynerOrg/xyter/pullRequest?prid=9008883)
```suggestion
	else {
```